x2gbfs/providers/voi_raumobil.py

```diff
@@ -43,7 +43,7 @@ class VoiRaumobil(BaseProvider):
             'max_range_meters': 10000,
             'name': 'Scooter',
             'wheel_count': 2,
-            'return_type': 'free_floating',
+            'return_constraint': 'free_floating',
```

Comment:
Is the return_contraint useful for Scooters? I expect that they all are free_floating.

Reply:
IMHO it does no harm to state this explicitly.